apps/spruce/src/pages/task/taskTabs/TaskHistory/TaskTimeline/DateSeparator/index.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { palette } from "@leafygreen-ui/palette";
+import { size } from "@evg-ui/lib/constants/tokens";
+
+const { gray } = palette;
+
+export const DATE_SEPARATOR_WIDTH = 8;
+interface DateSeparatorProps {
+  date?: Date | null;
+}
+const DateSeparator: React.FC<DateSeparatorProps> = ({ date }) => {
+  const parsedDate = new Date(date || "");
+  const formattedDate = parsedDate?.toLocaleDateString("en-US", {
+    month: "numeric",
+    day: "2-digit",
+  });
+  const dateString = formattedDate || "";
+  return (
+    <Container aria-label="date-separator" className="date-separator">
+      <StyledBadge className="date-badge">{dateString}</StyledBadge>
+      <Dot className="dot" />
+      <Line />
+    </Container>
+  );
+};
+
+const Container = styled.div`
+  width: ${DATE_SEPARATOR_WIDTH}px;
```

Comment:
we'd probably need to increase the width as I think it's currently too small to offer affordance when hovering to see the date